inference-engine/tests/functional/plugin/shared/include/single_layer_tests/proposal_invalid.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Looks like behavior test case. Could you move it?

Reply:
Moved